native/src/routingConfiguration.h

```diff
@@ -15,11 +16,25 @@ struct RoutingRule {
     string type;
 };
 
+struct DirectionPoint {
+    double distance = DBL_MAX;
```

Comment:
Easier to use value < 0 (-1) and avoid 'float.h' that could be possibly platform dependent